homeassistant/components/zwave_js/cover.py

```diff
@@ -65,27 +56,25 @@ class ZWaveCover(ZWaveBaseEntity):
     """Representation of a Z-Wave Cover device."""
 
     @property
-    def is_closed(self) -> bool:
+    def is_closed(self) -> Any:
```

Comment:
This should return a boolean. If mypy complains we can copy the expression to that explicitly with `bool`.

Reply:
Yup mypy gives me: `Returning Any from function declared to return "bool"`